pages/en/lb4/Controllers.md

```diff
@@ -156,3 +156,52 @@ class HelloController {
 - A `Repository` is LoopBack's database abstraction. See [Repositories](Repositories.html) for more.
 - `@get('/messages')` creates the `Route` for the Operation using `app.route()`.
 - `@param.query.number` adds a `number` param with a source of `query`.
+
+## Handling Errors in Controllers
+
+In order to specify errors for controller methods to throw, the class `HttpErrors` is used. `HttpErrors` can be found in the `@loopback/rest` package.
+
+The example below shows how `HttpErrors` can be used inside a controller method and how it can be tested.
+
+```js
+// the test
+import {HelloController} from 'path.to.controller'
+import {expect} from '@loopback/testlab'
+// ...
+
+// ...
+describe('Hello Controller', () => {
+  // ...
+  it('returns 412 Precondition Failed for non-positive limit', () => {
```

Comment:
Whoa! I don't think I've ever used 412 in my life!
...I like it.

Reply:
I'd change it to be a 422 Unprocessable Entity. The syntax is correct, but the _semantics_ of a request whose limit is a non-positive integer are incorrect, so it'd be the best match, IMO.

That being said, there's still debate about whether or not people should use 400 for that purpose, so either one is cool with me.